shm/inc/ship.hpp

```diff
@@ -6,9 +6,9 @@ class Ship {
 public:
     Ship()
         : id_(-1) {}
-    Ship(int capacity, int maxCrew, int speed, const std::string& name, size_t id)
+    Ship(int capacity, int maxCrew, int speed, const std::string& name, unsigned int id)
         : capacity_(capacity), maxCrew_(maxCrew), crew_(0), speed_(speed), name_(name), id_(id) {}
-    Ship(int maxCrew, int speed, size_t id)
+    Ship(int maxCrew, int speed, unsigned int id)

```

Comment:
Or we can left as it is and change type of `id_` in line 31. Also, shouldn't we change other types in C-Tor for to be consistent with: 
```  
    private:
    size_t capacity_;
    size_t maxCrew_;
    size_t crew_;
    size_t speed_;
    std::string name_;
    const int id_;
```

Reply:
It can't be `size_t` in that line because default constructor makes it equal to `-1`